content/monitors/faq/why-wont-my-monitor-trigger-when-the-agent-does-not-report.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
this one is already here: https://docs-staging.datadoghq.com/gus/redesign/integrations/faq/why-won-t-my-monitor-trigger-when-the-agent-does-not-report/

But your faq is in the right place, I'm going to delete the one in integrations